mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoMethodGenerator.java

```diff
@@ -215,6 +216,21 @@ protected void maybeAddTimestamp(String timestamp, MethodSpec.Builder methodBuil
     }
   }
 
+  protected void populateBuilderWithStatementAttribute(
+      MethodSpec.Builder builder, WithStatementAttributes withStatementAttributes) {
```

Comment:
1. `methodElement` is accessible from this class, so maybe you can extract the annotation here instead of passing it as a parameter.
2. nit: should be populateBuilderWithStatementAttribute**s** to be consistent